classes/Link.php

```diff
@@ -986,30 +986,30 @@ public function getMatchingUrlShopId()
      *
      * @return string
      */
-    public function getImageLink($name, $ids, $type = null)
+    public function getImageLink($name, $ids, $type = null, string $extension = '.jpg')
```

Comment:
```suggestion
    public function getImageLink($name, $ids, $type = null, string $extension = 'jpg')
```
Maybe the extension shouldn't include the dot `.` this could make the use of this method easier, you can loop through image formats like `['jpg', 'png']` or get a configuration `$ext = Configuration::get('DEFAULT_IMAGE_EXTENSION)` for example which are more likely to contain the format with the prefixed dot

I think it would improve the developer experience (no need to append a dot each time the method is used), but it's just a suggestion not mandatory